### ISSUE TITLE
Aprire l'allegato invece che effettuarne il download

### ISSUE THREAD

**Issue by danilomariotti** [pcrt/joomla-italia-theme#40]
Durante l'inserimento di una allegato (ad esempio PDF) tramite il media manager, viene aggiunto automaticamente l'attributo download="" che, al click sul link, provoca l'apertura da parte del browser della finestra per il salvtaggio del file invece di tentare di aprirlo direttamente, come sarebbe molto più utile per la maggior parte degli allegati che sono in formato PDF.
Esiste la possibilità di configurare joomla in modo da modificare questo comportamento del media manager, invece di dover andare a modificare ogni volta il codice html (penso sempre agli utenti non esperti che caricheranno contenuti)? 
Grazie

**Comment by Knock-Out** [comment]
Joomla! non ha una funzione integrata per modificare il comportamento predefinito del Media Manager quando si aggiunge un allegato.

Potresti risolvere il problema modificando il codice manualmente o utilizzando un'estensione di terze parti che consenta di personalizzare l'attributo "download" per i file allegati.

**Comment by pcrt** [comment]
non so se ho ben capito la domanda ma nel media managar di joomla puoi scegliere se far scaricare l'allegato o aprirlo direttamente
![Screenshot 2023-06-30 alle 16 19 46](https://github.com/pcrt/joomla-italia-theme/assets/44270932/aebfd445-a9df-4e4d-ba07-cef7fadb21aa)

**Comment by danilomariotti** [comment]
No, "Incorpora" lo inserisce/visualizza direttamente all'interno dell'articolo. La modalità giusta sarebbe Scarica ma viene inserito anche l'attributo download sche fa aprire la finestra di salvataggio.
Però ho notato che l'utilizzo di acf (advanced custom field) di Tassos marinos che hai suggerito nell'altra issue potrebbe cambiare qualcosa: il link che mi hai mandato apre proprio il pdf dentro al browser seza chiederne il salvataggio anche che è presente l'attributo download..... Faccio qualche verifica in più....
